docs/20newgroups.md

```diff
@@ -1,13 +1,70 @@
-# 20 Newsgroup Replication
+# 20Newsgroups Replication
+We're going to perform text classification using scikit on 20Newsgroups dataset.
 
-Download the pre-process dataset & lucene index
+##Data Prep
+
+We're going to use the repository's root directory as the working directory.
+There are many versions of the 20 Newsgroups dataset available on the web, we're specifically going to use [this one](http://qwone.com/~jason/20Newsgroups/) (the "bydate" version).
+### Download and extract dataset
+Please refer to instructions in [Anserini 20newsgroups replication](https://github.com/castorini/anserini/blob/master/docs/experiments-20newsgroups.md#data-prep)
+
+###Download and extract pre-built lucene index
+You can either follow instructions in [Anserini 20newsgroups replication](https://github.com/castorini/anserini/blob/master/docs/experiments-20newsgroups.md#indexing) and copy index files under `pyserini/indexes/20newsgroups`, or use our prebuilt index using following commands. 
+```bash
+mkdir indexes/20newsgroups
+wget https://www.dropbox.com/s/qo2wt6fzu01yt4c/lucene-index.20newsgroups.all.tar.gz -P indexes/20newsgroups
+tar xvfz indexes/20newsgroups/lucene-index.20newsgroups.all.tar.gz -C indexes/20newsgroups
 ```
-sh bin/get-20newsgroups-data.sh
+To confirm, `lucene-index.20newsgroups.all.tar.gz` should have MD5 checksum of `89ed27a08e3e77c51a9f1c28f0705da0`.
```

Comment:
keep this... in the end, say something like, here's the script that have everything put together...